### PR TITLE
[Bugfix] fix phase filtering

### PR DESCRIPTION
**Pull request by awegsche**
If the phase matrix is filtered, both rows **and** columns have to be filtered.

### CODE REVIEW

**Review by JoschD — APPROVED** [review]

**Review by fsoubelet — DISMISSED** [review]

**Review by tpersson — DISMISSED** [review]
Excellent! and we have seen that this indeed solve the issue!

**Review by fsoubelet — APPROVED** [review]
Updated version and changelog so we can release asap. Will merge once tests pass since everyone already approved.

**Pull request merged**
